src/tests/install_update_test.ts

```diff
@@ -0,0 +1,223 @@
+  },
+});
+
+const MOCK_SLACK_JSON_FILE = new TextEncoder().encode(MOCK_SLACK_JSON);
+const MOCK_IMPORT_MAP_FILE = new TextEncoder().encode(MOCK_IMPORT_MAP_JSON);
+
+Deno.test("update hook tests", async (t) => {
+  await t.step("createUpdateResp", async (evT) => {
+    await evT.step(
+      "if import_map.json has available updates, then response includes those updates",
+      async () => {
+        mockFile.prepareVirtualFile(
+          "./slack.json",
+          MOCK_SLACK_JSON_FILE,
+        );
+
+        mockFile.prepareVirtualFile("./import_map.json", MOCK_IMPORT_MAP_FILE);
+        mockFile.prepareVirtualFile("./slack.json", MOCK_SLACK_JSON_FILE);
+
+        const expectedHooksUpdateSummary = [{
+          name: "deno_slack_hooks",
+          current: "0.0.9",
+          latest: "0.0.10",
+        }];
+
+        const expectedImportsUpdateSummary = [
+          {
+            name: "deno_slack_sdk",
+            current: "0.0.6",
+            latest: "0.0.7",
+          },
+          {
+            name: "deno_slack_api",
+            current: "0.0.6",
+            latest: "0.0.7",
+          },
+        ];
+
+        assertEquals(
+          await createUpdateResp(MOCK_RELEASES),
+          {
+            name: SDK_NAME,
+            updates: [
+              ...expectedImportsUpdateSummary,
+              ...expectedHooksUpdateSummary,
+            ],
+          },
+          "correct update response not returned",
+        );
+      },
+    );
+  });
+
+  await t.step("updateDependencyFile", async (evT) => {
+    await evT.step(
+      "if dependency updates are available, the correct update summary is returned",
+      async () => {
+        const expectedHooksUpdateResp = [
+          {
+            name: "deno_slack_hooks",
+            current: "0.0.9",
+            latest: "0.0.10",
+          },
+        ];
+
+        const expectedImportsUpdateResp = [
+          {
+            name: "deno_slack_sdk",
+            current: "0.0.6",
+            latest: "0.0.7",
+          },
+          {
+            name: "deno_slack_api",
+            current: "0.0.6",
+            latest: "0.0.7",
+          },
+        ];
+
+        mockFile.prepareVirtualFile(
+          "./slack.json",
+          MOCK_SLACK_JSON_FILE,
+        );
+
+        mockFile.prepareVirtualFile(
+          "./import_map.json",
+          MOCK_IMPORT_MAP_FILE,
+        );
+
+        assertEquals(
+          await updateDependencyFile("./slack.json", MOCK_RELEASES),
+          expectedHooksUpdateResp,
+          "correct dependency update response for slack.json was not returned",
+        );
+
+        assertEquals(
+          await updateDependencyFile("./import_map.json", MOCK_RELEASES),
+          expectedImportsUpdateResp,
+          "correct dependency update response for import_map.json was not returned",
+        );
+      },
+    );
+
+    await evT.step(
+      "if file isn't found, an empty update array is returned",
```

Comment:
My expectation would be that if a file is not found, we error out noisily rather than silently continue. If folks agree, then let's change this test.